baseline/sweep

```diff
@@ -14,25 +14,37 @@ readonly PATIENCE=12
 readonly SED_EM_ITERATIONS=10
 readonly MAX_ENSEMBLE_SIZE=10
 
+LANGS=""
```

Comment:
One last question: shouldn't this be defined lower? E.g., right before the loop on line 112 or so?

Reply:
it's referenced in the functions so throws an unbound variable error

Reply:
Ahh okay makes sense.